### PR TITLE
Add Cloud/DevOps Engineer job description

### PR DESCRIPTION
**Pull request by duncanmadetech**

### CODE REVIEW

File: roles/cloud_devops_engineer.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Cloud / DevOps Engineer
+
+Bristol, UK.
+
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Senior Cloud Engineers deliver digital, data and technology outcomes that improve society. They do this by architecting and delivering cloud solutions, and coaching others to do so both within Made Tech and in public sector organisations.
```
Comment (rboww) [inline roles/cloud_devops_engineer.md:7]: Replace all "Senior Cloud Engineer" with Cloud Engineer

File: roles/cloud_devops_engineer.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+In Bristol, we are looking for Senior Cloud Engineers with a love for cloud, automation, slick tooling, and empowering software delivery through coaching teams. While we are looking for cloud engineers that are able to demonstrate significant expertise in the design and execution of cloud-based solutions, we expect all of our engineers to have a solid grasp of software engineering fundamentals.
```
Comment (rboww) [inline roles/cloud_devops_engineer.md:9]: Remove "In Bristol" given HMRC's policies we can look wider.

File: roles/cloud_devops_engineer.md (hunk 1 above)
Comment (duncanmadetech) [reply roles/cloud_devops_engineer.md:7]: Should it be "Our Cloud & DevOps Engineers" ?

File: roles/cloud_devops_engineer.md (hunk 2 above)
Comment (duncanmadetech) [reply roles/cloud_devops_engineer.md:9]: Should I remove the "Bristol, UK." from line 3 then? Or add other regions in? Talent will create a workable job entry per region.

File: roles/cloud_devops_engineer.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+# Cloud / DevOps Engineer
+
+Bristol, UK.
```
Comment (duncanmadetech) [inline roles/cloud_devops_engineer.md:3]: @rboww should I remove/add regions here?

File: roles/cloud_devops_engineer.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+* 💷 [Transparent Salary Bands](../roles/README.md) – We publish salary bands so you know you're being fairly compensated
+* 👌 [Annual Salary Reviews](../guides/compensation/salary_reviews.md) – We review your salary on an annual basis
+* ⛷️ [Pension Scheme](../benefits/pension_scheme.md) – We provide a pension scheme so you can save for your future and we'll contribute to it
+* 🚄 [Season Ticket Loan](../benefits/season_ticket_loan.md) – We provide loans to help you pay for your travel
+* 🚲 [Cycle To Work Scheme](../benefits/cycle_to_work_scheme.md) – We offer the cycle to work scheme to help pay for your bicycle
+* 🚕 [Expenses Paid](../guides/compensation/expenses.md) – Taxi to a meeting? Want to take a customer to lunch? Expenses are no hassle!
+
+## Salary
+
+- Bristol: £45,900-£81,000
```
Comment (duncanmadetech) [inline roles/cloud_devops_engineer.md:104]: @rboww Should I add/remove regions here?

File: roles/cloud_devops_engineer.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
+# Cloud / DevOps Engineer
+
+Bristol, UK.
+
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Cloud Engineers deliver digital, data and technology outcomes that improve society. They do this by architecting and delivering cloud solutions, and coaching others to do so both within Made Tech and in public sector organisations.
+
+We are looking for Cloud Engineers with a love for cloud, automation, slick tooling, and empowering software delivery through coaching teams. While we are looking for cloud engineers that are able to demonstrate significant expertise in the design and execution of cloud-based solutions, we expect all of our engineers to have a solid grasp of software engineering fundamentals.
```
Comment (duncanmadetech) [inline roles/cloud_devops_engineer.md:9]: inconsistent capitalisation "c/Cloud e/Engineers"

**Pull request merged**